syncstorage/storage/sql/queries_sqlite.py

```diff
@@ -22,14 +22,20 @@
 
 # Use the correct timestamp-handling functions for sqlite.
 
-PURGE_SOME_EXPIRED_ITEMS = "DELETE FROM %(bso)s "\
-                           "WHERE ttl < (strftime('%%s', 'now') - :grace) "
+PURGE_SOME_EXPIRED_ITEMS = """
+    DELETE FROM %(bso)s
+    WHERE ttl < (strftime('%%s', 'now') - :grace)
+"""
 
-PURGE_BATCHES = "DELETE FROM batch_uploads WHERE batch < " \
-                "   (SELECT strftime('%%s', 'now') - :grace) * 1000"
+PURGE_BATCHES = """
+    DELETE FROM batch_uploads
+    WHERE batch < (strftime('%s', 'now') - :lifetime - :grace) * 1000
```

Comment:
There was a subtle bug here, the '%%s' was incorrect because this string is never run through the string formatter, unlike the other two which include '%(bui)s' table references.  It should not escape the percent sign.